app/helpers/scheduled_pickups_helper.rb

```diff
@@ -1,14 +1,5 @@
 module ScheduledPickupsHelper
   def format_pickup_time(scheduled_pickup)
-    start_at = scheduled_pickup.start_at
-    end_at = scheduled_pickup.end_at
-    time_format = "%l:%M %P"
-
-    t(
-      "scheduled_pickups.format",
-      weekday: Weekday.find(start_at.wday).label,
-      start_at: start_at.strftime(time_format),
-      end_at: end_at.strftime(time_format),
-    )
+    TimeRange.new(scheduled_pickup.start_at, scheduled_pickup.end_at)
```

Comment:
@kenyonj yes, when interpolated, `#to_s` would turn it into the formatted string.